modules/manage/partials/authentication.adoc

```diff
@@ -1102,6 +1105,7 @@ redpanda:
       key_file: mtls_broker.key
       cert_file: mtls_broker.crt
       truststore_file: mtls_ca.crt
+      crl_file: mtls_ca.crl # Optional
```

Comment:
@michael-redpanda, I'm not sure whether the updates from here to the end of the page are required??

Reply:
I'd say no